examples/meteringlab/c1/cluster.yaml

```diff
@@ -0,0 +1,94 @@
+# Namespace where the Scylla Cluster will be created
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: scylla
+
+---
+
+# Scylla Cluster
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  name: scylla-cluster
+  namespace: scylla
+spec:
+  version: 5.0.5
+  agentVersion: 3.0.1
+  cpuset: true
+  network:
+    hostNetworking: true
+  sysctls:
+  - "fs.aio-max-nr=2097152"
+  datacenter:
+    name: us-east-1
+    racks:
+    - name: a
+      members: 2
+      storage:
+        storageClassName: local-raid-disks
+        capacity: 1800G
+      resources:
+        limits:
+          cpu: 3
+          memory: 24G
+      placement:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                - key: topology.kubernetes.io/zone
+                  operator: In
+                  values:
+                    - us-east-1a
+        tolerations:
+          - key: role
+            operator: Equal
+            value: scylla-clusters
+            effect: NoSchedule
+    - name: b
+      members: 2
+      storage:
+        storageClassName: local-raid-disks
+        capacity: 1800G
+      resources:
+        limits:
+          cpu: 3
+          memory: 24G
```

Comment:
for each scylla clsuter you have to create a configmap within the same namespace:
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: xfs-nvme-249g-iotune
  namespace: {{ $ns }}
data:
  io_properties.yaml: |
    disks:
    - mountpoint: /var/lib/scylla
      read_iops: 385400
      read_bandwidth: 3087631104
      write_iops: 240628
      write_bandwidth: 2289281280
```
and add these to every rack:
```
      volumeMounts:
      - mountPath: /etc/scylla.d/io_properties.yaml
        name: iotune
        subPath: io_properties.yaml
      volumes:
      - name: iotune
        configMap:
          name: xfs-nvme-249g-iotune